nomt/src/beatree/leaf/node.rs

```diff
@@ -41,7 +41,7 @@ pub const MAX_LEAF_VALUE_SIZE: usize = (LEAF_NODE_BODY_SIZE / 3) - 32;
 /// The maximum number of node pointers which may appear directly in an overflow cell.
 ///
 /// Note that this gives an overflow value cell maximum size of 100 bytes.
-pub const MAX_OVERFLOW_CELL_NODE_POINTERS: usize = 23;
+pub const MAX_OVERFLOW_CELL_NODE_POINTERS: usize = 15;
```

Comment:
Does this require updating the comment above?

Reply:
No. 15*4 + 32 + 8 = 100 still.